catalog/ACA/ACA_reef_habitat_v2_0.jsonnet

```diff
@@ -0,0 +1,322 @@
+local id = 'ACA/reef_habitat/v2_0';
```

Comment:
Please remove trailing space characters.  Some text editors can do this.  Or you can some somethings like:

```shell
perl -pi -e 's/\s+\n/\n/g' catalog/ACA/ACA_reef_habitat_v2_0.jsonnet
```

Reply:
OK, done as best i can tell